src/main/java/org/jabref/logic/ai/ingestion/GenerateEmbeddingsForSeveralTask.java

```diff
@@ -79,7 +80,8 @@ public Void call() throws Exception {
         linkedFiles
                 .stream()
                 .map(processingInfo -> {
-                    processingInfo.setState(ProcessingState.PROCESSING);
+                    UiTaskExecutor.runInJavaFXThread(() -> processingInfo.setState(ProcessingState.PROCESSING));
```

Comment:
This is the wrong place, see my other PR

Reply:
https://github.com/JabRef/jabref/pull/11796

Reply:
Hmmm, I see.

But I had an exception on this line. Maybe I'll investigate the bug further later. I'll just focus on group chat history in this PR

Reply:
The exception comes because we access the UI, but at this point we are in "logic" and we cannot use UI In logic, thus we need to go the point in the UI where the notification is triggered.